<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Elija tipo de lomito:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rb_carne">
         <property name="text">
          <string>Carne - $500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_pollo">
         <property name="text">
          <string>Pollo - $450</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_cerdo">
         <property name="text">
          <string>Cerdo - $400</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Elija extras</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="cb_huevo">
         <property name="text">
          <string>Huevo frito - $100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_papas">
         <property name="text">
          <string>Papas fritas - $200</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_gaseosa">
         <property name="text">
          <string>Gaseosa - $250</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_delivery">
         <property name="text">
          <string>Delivery - $100</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Precio Total:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_calcular">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
